debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -712,9 +712,16 @@ private static int validateDatabaseExcludeList(Configuration config, Field field
 
     private static int validateMessageKeyColumnsField(Configuration config, Field field, Field.ValidationOutput problems) {
         String msgKeyColumns = config.getString(MSG_KEY_COLUMNS);
-        if (msgKeyColumns != null && !MSG_KEY_COLUMNS_PATTERN.asPredicate().test(msgKeyColumns)) {
-            problems.accept(MSG_KEY_COLUMNS, msgKeyColumns, MSG_KEY_COLUMNS.name() + " has an invalid format (expecting '" + MSG_KEY_COLUMNS_PATTERN.pattern() + "')");
-            return 1;
+        if (msgKeyColumns != null) {
+            if (msgKeyColumns.contains(";")) {
+                msgKeyColumns = msgKeyColumns.replaceAll(";+$", "");
+            }
+            for (String substring : msgKeyColumns.split(";")) {
```

Comment:
Let's extract a pattern instance and re-use it:

    private static final Pattern COLUMN_SPLIT = Pattern.compile(";");

And here: `COLUMN_SPLIT.split(msgKeyColumns) ...`.